charts/nd-common/templates/_monitors.tpl

```diff
@@ -90,3 +93,38 @@ spec:
       {{- end }}
 {{- end }}
 {{- end }}
+
+{{/*
+
+This function creates Prometheus recording rules that are useful to include in
+other queries about the configuration of this chart.
+
+*/}}
+
+{{- define "nd-common.monitorRules" }}
+{{- if .Values.monitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "nd-common.fullname" . }}
+  {{- with .Values.monitor.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "nd-common.labels" . | nindent 4 }}
+    {{- with .Values.monitor.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  groups:
+  - name: {{ include "nd-common.fullname" . }}.monitorRules
+    rules:
+      - record: namespace:enforced_sample_limit
```

Comment:
I see why you chose `namespace:`... but this is not a namespace-specific limit, instead its actually a `PodMonitor` specific limit. Since the current _enforced_ value is recorded as `prometheus_operator_prometheus_enforced_sample_limit`, I suggest `pod_monitor_sample_limit`.

Reply:
Changed.